crabs/tracker/evaluate_tracker.py

```diff
@@ -41,35 +44,40 @@ def create_gt_list(
                     data["y"],
                     data["x"] + data["width"],
                     data["y"] + data["height"],
-                    data["id"],
                 ],
                 dtype=np.float32,
             )
+            track_id = np.array([data["id"]], dtype=np.float32)
+
             if gt_boxes_list[frame_number].size == 0:
                 gt_boxes_list[frame_number] = bbox.reshape(
                     1, -1
                 )  # Initialize as a 2D array
+                gt_ids_list[frame_number] = track_id
             else:
                 gt_boxes_list[frame_number] = np.vstack(
                     [gt_boxes_list[frame_number], bbox]
                 )
-        return gt_boxes_list
+                gt_ids_list[frame_number] = np.hstack(
+                    [gt_ids_list[frame_number], track_id]
+                )
+
+        return gt_boxes_list, gt_ids_list
 
-    def get_ground_truth_data(self) -> list[np.ndarray]:
+    def get_ground_truth_data(
```

Comment:
I think the conclusion is: let's use the frame numbers as they are in the csv (whatever base they are in) rather than frame indices (0-based), and use a dict rather than lists. 

That should make us agnostic to the convention used in the csv (1-based or 0-based, starting in the first frame or not...)